src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -1612,3 +1621,144 @@ def cli_cosmosdb_collection_update(client,
 
         result['offer'] = client.ReplaceOffer(offer['_self'], offer)
     return result
+
+
+def cli_cosmosdb_sql_role_definition_create(client,
+                                            resource_group_name,
+                                            account_name,
+                                            role_definition_body):
+    '''Creates an Azure Cosmos DB SQL Role Definition '''
+    role_definition_create_resource = SqlRoleDefinitionCreateUpdateParameters(
+        role_name=role_definition_body['RoleName'],
+        type=role_definition_body['Type'],
+        assignable_scopes=role_definition_body['AssignableScopes'],
+        permissions=role_definition_body['Permissions'])
+
+    return client.begin_create_update_sql_role_definition(role_definition_body['Id'], resource_group_name, account_name, role_definition_create_resource)
```

Comment:
with support_no_wait=True, you also need to use sdk_no_wait for all your long running operation functions. e.g. 
https://github.com/Azure/azure-cli/blob/4a5d74215d5d1566f839c2d11fd360d300f56b62/src/azure-cli/azure/cli/command_modules/network/custom.py#L260

Reply:
Please add a test for command with `--no-wait` to make sure it work as expected.